### PR TITLE
refactor: use alembic migrations to setup up-to-date database for tests in 'taxonomy'

### PR DESCRIPTION
**Pull request by spiridonovfed**

### CODE REVIEW

File: taxonomy/app/database.py [hunk 1]
```diff
@@ -17,6 +18,16 @@
...
 engine = create_engine(SQLALCHEMY_DATABASE_URL)
 
+# Ensure LTREE extensions is installed
+with engine.connect() as conn:
+    try:
+        conn.execute(sqlalchemy.sql.text("CREATE EXTENSION IF NOT EXISTS ltree"))
+    except sqlalchemy.exc.ProgrammingError as err_:
+        # Exctension installed, just skip error
+        if "DuplicateObject" not in str(err_):
+            raise err_
```
Comment (cakeinsauce) [inline taxonomy/app/database.py:28]: May we move it to some sort of function? and add it as pre-setup step for app?

Comment (cakeinsauce) [reply taxonomy/app/database.py:28]: For instance, if it's not done already, create setup_db function to adjust all db settings before starting the app

Comment (spiridonovfed) [reply taxonomy/app/database.py]: I've decided to move it directly to alembic migrations as long as it set our database up and it seems appropriate having this logic there

**Review by cakeinsauce — APPROVED** [review]
LGTM!
Squash commits before merging, please

**Pull request merged**